src/components/SidebarError/SidebarError.scss

```diff
@@ -60,9 +60,14 @@
           color: var(--sa-neutral-color-3);
         }
       }
-      .sidebar-error-button {
+      .sidebar-error-buttons{
+        display: flex;
         text-align: center;
-        margin-bottom: 13px;
+        margin-bottom: 16px;
+        justify-content: flex-start;
```

Comment:
I would change justify-content to center, it looks a bit nicer. 
(idk i just like things to be simetrical, but if you like it this way just leave it as it is :) )

Reply:
Made the changes. It was supposed to be centered according to the design, but it looked weird when the sidebar was resized to the max.

Reply:
Also cleared the text field input on reset